### PR TITLE
PHP 8 Prototype

### PR DESCRIPTION
**Pull request by derrabus**
I have updated the demo application to showcase the new possibilities with php 8 and Symfony 5.2. These changes are of course too bleeding edge to merge, but you might get a good first impression about how a Symfony app on php 8 might look like.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -3,40 +3,43 @@
...
     "replace": {
         "symfony/polyfill-php70": "*",
-        "symfony/polyfill-php72": "*"
+        "symfony/polyfill-php72": "*",
+        "symfony/polyfill-php73": "*",
+        "symfony/polyfill-php80": "*"
     },
     "require": {
-        "php": "^7.2.9",
+        "php": ">= 7.4",
```
Comment (derrabus) [inline composer.json:15]: ⚠️ Note: This is a lie. Not all of our dependencies oficially support PHP 8 yet, so we pretend to pull packages for PHP 7.4.

**Comment by shyim** [comment]
Did you get opcache preload running? I am getting segsev on each request

**Comment by derrabus** [comment]
@shyim I haven't tried, tbh.

File: src/Controller/Admin/BlogController.php [hunk 2]
```diff
@@ -64,13 +63,16 @@ public function index(PostRepository $posts): Response
...
      * NOTE: the Method annotation is optional, but it's a recommended practice
      * to constraint the HTTP methods each controller responds to (by default
      * it responds to all methods).
      */
+<<<<<<< HEAD
     public function new(Request $request, EntityManagerInterface $entityManager): Response
+=======
+    #[Route('/new', methods: ['GET', 'POST'], name: 'admin_post_new')]
+    public function new(Request $request): Response
+>>>>>>> 44c4fc1 (PHP 8 Prototype)
```
Comment (stof) [inline src/Controller/Admin/BlogController.php:75]: This is a broken conflict resolution

File: symfony.lock [hunk 3]
```diff
@@ -44,6 +44,9 @@
     "doctrine/deprecations": {
         "version": "v0.5.3"
     },
+    "doctrine/deprecations": {
+        "version": "v0.5.3"
+    },
```
Comment (derrabus) [inline symfony.lock:49]: Should be reverted because the block is redundant now: see the three lines above it.

Comment (javiereguiluz) [reply symfony.lock:49]: Done in https://github.com/symfony/demo/pull/1277/commits/8c4defe7556f4a08cf95d2a5d1ad977a5c2c47a2. Thanks!

**Pull request merged**